std_msgs/Header header

## rigid-body position and velocity
float32[3] position         ## [m]
float32[3] velocity         ## [m/s]
float32[3] angular_velocity ## [rad/s]
float32[4] quaternion       ##
